DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/NetworkProtectionTunnelController.swift

```diff
@@ -382,7 +382,7 @@ final class NetworkProtectionTunnelController: TunnelController, TunnelSessionPr
 
     public var session: NETunnelProviderSession? {
         get async {
-            guard let manager = await manager,
+            guard let manager = internalManager,
```

Comment:
The difference between manager and `internalManager` is that `internalManager` is our cached configuration.  This is invalidated if deleted so it's fine to use it and we avoid unnecessary calls to load all configurations.